app/models/middleware_server.rb

```diff
@@ -42,11 +43,11 @@ def in_domain?
 
   # Returns whether a server is immutable or not.
   #
-  # By default, we define all servers as being mutable, so that power
-  # operations are allowed, and it is the provider responsability to
+  # By default, we define all servers as being immutable, so that power
+  # operations are not allowed, and it is the provider responsability to
   # reimplement this method if necessary.
   def immutable?
-    properties['Immutable'] == 'true'
+    true
```

Comment:
Is this change related to the adding of STI types?  Seems bolted on, and should be a separate PR, but maybe I'm missing the rationale for it here?